.github/workflows/main.yml

```diff
@@ -14,11 +14,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
+      - name: Setup JDK 11
```

Comment:
I believe the prior spelling was actually correct, given that that "set up" is used as a verb here rather than as a noun, in which case your spelling would have been correct? See https://www.merriam-webster.com/dictionary/setup.

Reply:
Oh yes, I agree.